CUETools.Codecs.FlaCuda/FlaCudaWriter.cs

```diff
@@ -2055,7 +2055,7 @@ struct FlakeEncodeParams
 
 		public bool do_seektable;
 
-		public int flake_set_defaults(int lvl, bool encode_on_cpu)
+		public int Flake_set_defaults(int lvl, bool encode_on_cpu)

```

Comment:
Please undo. The lower case letter at the beginning of the method name should stay, like in other methods.